sway/input/input-manager.c

```diff
@@ -539,6 +541,18 @@ void input_manager_reset_all_inputs() {
 	wl_list_for_each(input_device, &server.input->devices, link) {
 		input_manager_reset_input(input_device);
 	}
+
+	// If there is at least one keyboard using the default keymap and repeat
+	// delay and rate, then it is possible that there is a keyboard group that
+	// needs to be disarmed. Since all inputs are being reset anyway, just
+	// disarm all of them.
+	struct sway_seat *seat;
+	wl_list_for_each(seat, &server.input->seats, link) {
+		struct sway_keyboard_group *group;
+		wl_list_for_each(group, &seat->keyboard_groups, link) {
+			sway_keyboard_disarm_key_repeat(group->seat_device->keyboard);
```

Comment:
This fixes the issue, thanks. One question is, should this also be calling `seat_configure_keyboard`? There are exit/re-enter events that it's not clear that this patch is sending (or, does it not matter for keyboard group keyboards?)

https://github.com/swaywm/sway/blob/726d187d3ce83431a2565cc1b423af99da5f7b6b/sway/input/seat.c#L700

Reply:
I don't think it's strictly needed now, but may be for #4975 - which is related to keyboards being added/removed to a keyboard group. 

I changed it to call `seat_reset_device`, which is the same method normal input devices use, instead of just `sway_keyboard_disarm_key_repeat`. This required adding a condition in `seat_configure_keyboard` to not call `sway_keyboard_configure` for keyboard group keyboards.